server/main.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import http from 'http';
+import https from 'https';
+
+import { loadConfig, watchConfig, getServerName } from 'serverConfig';
+import { returnExpress } from 'core';
+import { log } from 'placeholder';
+
+const errorHandler: (err: Error, ...others: unknown[]) => void = (
+  err,
+  ...others
+) => {
+  log(
+    serverName,
+    'runtime',
+    'main',
+    'Error',
+    `Error thrown: ${err.message}`,
+    err,
+    others
```

Comment:
From typing perspective, do we want to restrict it? Or are we likely to do `x.toString()` and end up with `[Object object]`?

Reply:
That will depend on the logger. Again, I wanted to 'shape' to be in place, with the logging issue to provide the implementation